### DISCUSSION TITLE
Welcome to biobankAccelerometerAnalysis Discussions!

### DISCUSSION THREAD

**Discussion by chanshing** [activityMonitoring/biobankAccelerometerAnalysis#217] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rcliffo6** [comment]
I am a happy user of the UK Biobank, and have accelerometer data as part of a study on balance in the elderly. My question is:
Is it possible to measure one of the axes, say to look at lateral sway, to see if this is correlated to either self-report of falls or to a participants' description of dizziness? If I look at overall activity (moderate, mild, etc.), that might tell me that the subject is not moving because they are dizzy, but it wouldn't measure the balance system. Is it possible to extract information say on the z-axis?

I understand that several accelerometers were used during the UKB study and that right hand versus left hand might be a factor - perhaps extract data separately? Thanks for any insight anyone might have.
Royce

**Reply by angerhang** [reply]
Hey @rcliffo6 thanks for reaching out and being patient on this topic.

It is possible to extract information that are axis-specific, x, y, and z for example.

The participants were always introduced on wear the device on their dominant-wrist which could be left or right.

There are perhaps better ways to measuring balancing from the devices in the UKB. Participants who reported to have falls might not actually fall during the device wear as there is a long lag between the questionnaire and the device wear. 

You could try to extract other metrics related to balancing such as looking as the variations or symmetry of gaits things like that.

**Reply by chanshing** [reply]
Hi @rcliffo6 Sorry for the slow response. For such a customized analysis, you could have a look at this package which will extract the raw x,y,z timeseries from the devices: https://github.com/OxWearables/actipy

**Comment by kaitsing** [comment]
Hello, is there a way to use this package with AGD files?

**Comment by aidendoherty** [comment]
Dear @kaitsing - we currently do not support AGD files and I don't think we have any plans to. My understanding is that these are from older actigraph devices that output summary epoch level data. Our software requires raw data as input from which to derive features for the machine learning behavioural inference.
